test.js

```diff
@@ -68,6 +68,14 @@ describe('chalk.enabled', function () {
 	});
 });
 
+describe('chalk.constructor', function () {
+	it('should create a isolated context where colors can be disabled', function () {
+		var ctx = new chalk.constructor(false);
```

Comment:
I don't think passing `false` into the constructor is very intuitive. Wouldn't it be better to just create a new instance and then change the `enabled` property?

```js
var ctx = new chalk.constructor();
ctx.enabled = false;
```